app/models/server/raw/Rooms.js

```diff
@@ -204,8 +204,8 @@ export class RoomsRaw extends BaseRaw {
 				},
 			],
 			name: nameRegex,
+			'u._id': uid,
```

Comment:
I know we used this filter on the other validation as well, but I don't think this is the correct way to validate what we need. this field actually stores the "creator" of the channel, which is formerly know as the owner, but we also have a owner role, which I think is more appropriate. Like when a user transfer the ownership of a channel we don't update the `u` field, we just give the new user the `owner` role.

the role is stored on Subscriptions, so here you could rely that filtered list of _ids where the user has the `owner` role..

Reply:
Yeah, I know about the `owner` role of the subs, I was trying to keep both filters the same (the one here and the one in the `addRooms` service.

If checking the subs is the way to go, I'll change this and the other :)

Reply:
yep, I think that is the expected behavior.. I didn't realize that before 🙈

Reply:
Yup, you're right on that 👀 we should've modified that validation before 😬

Reply:
@sampaiodiego what should be the behavior of the autocomplete then? Currently, it fetches all public channels and private channels the user is part of. Should this be changed to only show channels where the user is the owner (disregard if it's public/private)?

Reply:
Yep. I think so :)